### PR TITLE
Port iOS MongoDB examples page to example

### PR DESCRIPTION
**Pull request by cbush**
- Not as full-featured as it could be, but not worse than what we had originally.
- Page can be improved iteratively.

### Docs staging link (requires sign-in on MongoDB Corp SSO):
https://docs-mongodbcom-staging.corp.mongodb.com/realm/docsworker-xlarge/remote-mongodb/sdk/ios/examples/mongodb-remote-access

### CODE REVIEW

File: source/sdk/ios/examples/mongodb-remote-access.txt [hunk 1]
```diff
@@ -12,104 +12,37 @@ MongoDB Remote Access
...
+Example
+-------
 
-Count Documents in a Collection
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+You can access your linked data source using the currently logged-in
+user's :swift-sdk:`mongoClient
+<Extensions/User.html#/s:So7RLMUserC10RealmSwiftE11mongoClientySo08RLMMongoE0CSSF>`.
+The ``serviceName`` parameter is the name you gave the data source when
+linking it (by default, this is "mongodb-atlas").
```
Comment (nathan-contino-mongo) [inline source/sdk/ios/examples/mongodb-remote-access.txt:40]: I'm not sure that you can customize this any more, they seem to change the flow for this all the time.

File: source/sdk/ios/examples/mongodb-remote-access.txt [hunk 2]
```diff
@@ -12,104 +12,37 @@ MongoDB Remote Access
...
-
-Aggregation Stage: $addFields
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-
-.. _ios-aggregate-unwind:
-
-Aggregation Stage: $unwind
-~~~~~~~~~~~~~~~~~~~~~~~~~~
+.. literalinclude:: /examples/generated/code/start/AccessMongoDB.codeblock.remote-mongodb.swift
+   :language: swift
```
Comment (nathan-contino-mongo) [inline source/sdk/ios/examples/mongodb-remote-access.txt:48]: I guess you didn't introduce this example in this PR, but I wonder if the concept of using a user id as a partition might be a lot of heavy conceptual lifting for readers who aren't familiar with Sync. Perhaps when we revisit this page to add more examples, we can choose a first example that's a little more approachable for new users.

**Review by nathan-contino-mongo — APPROVED** [review]
LGTM

File: source/sdk/ios/examples/mongodb-remote-access.txt (hunk 1 above)
Comment (cbush) [reply source/sdk/ios/examples/mongodb-remote-access.txt:40]: You can link multiple data sources to your app, so I am certain you can still customize this name!

File: source/sdk/ios/examples/mongodb-remote-access.txt (hunk 2 above)
Comment (cbush) [reply source/sdk/ios/examples/mongodb-remote-access.txt:48]: Good point.

**Pull request merged**